volontulo/urls.py

```diff
@@ -41,6 +41,11 @@
         name='offer_form'
     ),
     url(
+        r'^offers/form/(?P<organization_id>[0-9]+)/(?P<offer_id>[0-9]+)$',
```

Comment:
Why we need to have organization-id when we want to edit particular offer fully identified by offer-id?